tests/components/__snapshots__/DigitalTerrainWebApp.test.jsx.snap

```diff
@@ -4,29 +4,253 @@ exports[`DigitalTerrainWebApp should render a default view 1`] = `
 <div
   class="digital-terrain-web-app"
 >
-  <header
-    class="digital-terrain-web-app-header"
+  <section
```

Comment:
This smells auto-generated... If it is, should it be included in the `.gitignore`?

Reply:
It is auto generated. These are snapshots rendered from snapshot tests `.toMatchSnapshot()` we'll use these to ensure our HTML DOMs are as we expect. The first commit of a snapshot isn't that important so long as you inspect it. But in the future if we add some UI and we run our tests snapshots should fail for where we added the UI components. If they don't we know that something isn't rendering the way we expect. Similarly, if we run the snapshots tests and they fail because something is something but we didn't change anything then we can track down what is causing that failure and revert it.

With that being said we'll need to commit these to ensure we keep them updated with the latest changes and Jest can use them to compare against.